detectron2/config/defaults.py

```diff
@@ -633,3 +633,14 @@
 # Do not commit any configs into it.
 _C.GLOBAL = CN()
 _C.GLOBAL.HACK = 1.0
+
+## WandB config
+_C.WANDB = CN()
+_C.WANDB.PROJECT_NAME = "Detectron2"
```

Comment:
Can you add an option here for `_C.WANDB.ENTITY` so that people using Teams can pass a team entity

Reply:
Yeah all wandb args options will be added in this